certora/conf/ERC20.conf

```diff
@@ -1,15 +1,15 @@
 {
   "files": [
-    "contracts/ERC20.sol:ERC20"
-  ],
-  "verify": [
-    "ERC20:certora/spec/ERC20.spec"
+    "contracts/ERC20TransferBug.sol:ERC20"
```

Comment:
oh I think you might want a separate conf?

Reply:
yes sorry my bad, I forgot I modified the original one